packages/notte-browser/src/notte_browser/window.py

```
@@ -256,7 +263,33 @@ def is_chromium_based(self) -> bool:
+    def _invalidate_cdp_session(self, tab_idx: int | None = None) -> None:
+        cdp_page = self.tabs[tab_idx] if tab_idx is not None else self.page
+        _ = self._cdp_sessions.pop(id(cdp_page), None)
+
+    async def _close_all_cdp_sessions(self) -> None:
+        sessions = list(self._cdp_sessions.values())
+        self._cdp_sessions.clear()
+        for session in sessions:
+            try:
+                _ = await asyncio.wait_for(session.detach(), timeout=2.0)
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)